rewrite-test/src/test/java/org/openrewrite/FindSourceFilesTest.java

```diff
@@ -76,7 +77,8 @@ void windows() {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"hello.txt", "/hello.txt", "\\hello.txt", "./hello.txt", ".\\hello.txt"})
+    @ValueSource(strings = {"hello.txt", "/hello.txt", "\\hello.txt", "./hello.txt", ".\\hello.txt", ""})
+    @NullSource
     void findRoot(String filePattern) {
```

Comment:
Two failures in this test it seems https://ge.openrewrite.org/s/akobtpkwdhh44/tests/overview?outcome=FAILED

Reply:
will check back over; was having issues with my local build the other day

Reply:
fixed; that's what I get for only skimming the test case I was adding on to :)

Reply:
All good, thanks for the quick fix!